src/databases/Xdmf/avtXdmfFileFormat.C

```diff
@@ -736,6 +754,22 @@ vtkDataSet * avtXdmfFileFormat::GetMesh(int timestate, int domain, const char *m
             break;
         case VTK_RECTILINEAR_GRID:
             dataSet = this->ReadRectilinearGrid(gridToRead);
+
+            // If we're here for a curve object, we adjust the rectgrid by ignoring
+            // Y and Z dimensions and then add the curve's "values" as a point data array.
+            // We do this by creating a new rectgrid and copying X coordinates over.
+            if (strcmp(_meshname, meshname))
+            {
+                vtkRectilinearGrid *oldGrid = vtkRectilinearGrid::SafeDownCast(dataSet);
+                vtkRectilinearGrid *newGrid = vtkRectilinearGrid::New();
+                newGrid->SetDimensions(oldGrid->GetDimensions()[0], 1, 1);
+                newGrid->SetXCoordinates(oldGrid->GetXCoordinates());
+                vtkDataArray *yvals = GetVar(timestate, domain, _meshname);
+                yvals->SetName(meshname);
```

Comment:
Should these be `_meshname` and `meshname` and not the same?

Reply:
Nope. Its correct as is. There is some name mapping going on above it.